packages/components/fields/select-field/src/select-field.tsx

```diff
@@ -308,12 +308,14 @@ export default class SelectField extends Component<TSelectFieldProps> {
 
     if (this.props.isMulti) {
       warning(
-        Array.isArray(this.props.value),
+        Array.isArray(this.props.value) ||
+          typeof this.props.value === 'undefined',
         'SelectField: `value` is expected to be an array of string when isMulti is true'
```

Comment:
```suggestion
        `SelectField: "value" is expected to be an array of string when "isMulti" is true, instead got ${this.props.value}.`
```

Reply:
Updated in 60d2ee841c61658c25251cd1b69ce397c929b0de

Thanks!